cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -174,8 +174,21 @@ private[akka] final class BootstrapCoordinator(discovery: ServiceDiscovery,
       discoverContactPoints()
 
     case ServiceDiscovery.Resolved(_, contactPoints) ⇒
-      log.info("Located service members based on: [{}]: [{}]", lookup, contactPoints.mkString(", "))
-      onContactPointsResolved(contactPoints)
+      val filteredContactPoints: Iterable[ResolvedTarget] =
+        if (lookup.portName.isDefined)
+          contactPoints
+        else
+          contactPoints.groupBy(_.host).flatMap {
+            case (host, immutable.Seq(singleResult)) =>
+              immutable.Seq(singleResult)
+            case (host, multipleResults) =>
+              println(s"Looking for ${settings.contactPoint.fallbackPort}")
+              multipleResults.filter(_.port.contains(settings.contactPoint.fallbackPort))
+          }
+
+      log.info("Located service members based on: [{}]: [{}], filtered to [{}]", lookup, contactPoints.mkString(", "),
```

Comment:
should this be info? done for each probing?

Reply:
It was already logged each time the probing endpoints are discovered (it just moved around a bit in this PR) and did come in handy when diagnosing problems, so I think it is nice to keep this at INFO at least for now.

Reply:
ok